client/packages/system/src/Item/Components/VaccineItemSelect/VaccineItemSelect.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  Checkbox,
+  AutocompleteOptionRenderer,
+  FnUtils,
+  AutocompleteMulti,
+  VaccineCourseItemNode,
+} from '@openmsupply-client/common';
+import { useVaccineItems } from 'packages/system/src/IndicatorsDemographics/api/hooks/document/useVaccineItems';
+import { DraftVaccineCourse } from 'packages/system/src/Immunisation/api/hooks/useVaccineCourse';
+
+interface VaccineItemSelectProps {
+  extraFilter?: (item: VaccineCourseItemNode) => boolean;
+  onChange: (newData: Partial<DraftVaccineCourse>) => void;
+  draft: DraftVaccineCourse;
+}
+
+const renderOption: AutocompleteOptionRenderer<VaccineCourseItemNode> = (
+  props,
+  option,
+  { selected }
+): JSX.Element => {
+  return (
+    <li {...props}>
+      <Checkbox checked={selected} />
```

Comment:
NIT - seeing as you have `filterSelectedOptions` enabled below I don't think the checkbox is needed - if anything I found it made me expect to open the list and see my selected options checked :)